tests/PSRule.Rules.Azure.Tests/Resources.AppConfig.json

```diff
@@ -396,14 +397,15 @@
       "encryption": {
         "keyVaultProperties": null
       },
-      "privateEndpointConnections": null
+      "privateEndpointConnections": null,
+      "enablePurgeProtection": true
```

Comment:
Let's have one case with `enablePurgeProtection` set to `false`.

Reply:
Agreed.